Apps/Sandcastle/gallery/Camera Limiter.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
```

Comment:
The thumbnail just looks like the hello world example when in Sandcastle- Can we make it look different? Maybe show the boxes more closely. Also make sure the thumbnail is the same dimensions as the rest.

Reply:
checked dimensions and updated the new photo